.github/workflows/config/spelling_allowlist.txt

```diff
@@ -62,6 +65,7 @@ canonicalizer
 canonicalizes
 codebase
 composable
+config
```

Comment:
This is not a word and should not be listed here. Is this used in a filename somewhere?